doc/pr2_tutorials/planning/scripts/move_group_python_interface_tutorial.py

```diff
@@ -192,17 +192,15 @@ def move_group_python_interface_tutorial():
   ## for the end-effector to go through.
   waypoints = []
 
-  # start with the current pose
-  waypoints.append(group.get_current_pose().pose)
-
   # first orient gripper and move forward (+x)
   wpose = geometry_msgs.msg.Pose()
   wpose.orientation.w = 1.0
-  wpose.position.x = waypoints[0].position.x + 0.1
-  wpose.position.y = waypoints[0].position.y
-  wpose.position.z = waypoints[0].position.z
+  wpose.position.x = wpose.position.x + 0.1
+  wpose.position.y = wpose.position.y
+  wpose.position.z = wpose.position.z
```

Comment:
This is odd that you are directly assigning the variable to itself. Instead:

wpose.position.x = 0.1
wpose.position.y = 0
wpose.position.z = 0